From a25879017434858dba46b617e770ca9330e189d1 Mon Sep 17 00:00:00 2001
From: maxice8 <thinkabit.ukim@gmail.com>
Date: Wed, 13 Dec 2017 01:26:13 -0200
Subject: [PATCH] libtrace/libtrace.c: add missing limits.h include

Fix build on Musl systems, tested on x86_64-musl Void Linux
---
 src/libtrace/libtrace.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libtrace/libtrace.c b/src/libtrace/libtrace.c
index 7bdf6c948..2b0a76938 100644
--- a/src/libtrace/libtrace.c
+++ b/src/libtrace/libtrace.c
@@ -23,6 +23,7 @@
 #include <string.h>
 #include <dlfcn.h>
 #include <sys/types.h>
+#include <limits.h>
 #include <unistd.h>
 #include <sys/socket.h>
 #include <netinet/in.h>
